src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -119,12 +119,44 @@ public async Task<JObject> TryToRunOnLoadedClasses(string varName, CancellationT
                     }
                 }
                 var store = await proxy.LoadStore(sessionId, token);
+
+                // first - check the current assembly
+                var currentFrame = ctx.CallStack.FirstOrDefault();
+                var currentAssembly = currentFrame.Method.Info.TypeInfo.assembly;
```

Comment:
You can create a attribute Namespace on TypeInfo class and set this value in the constructor then use it here.

Reply:
I implemented it :)